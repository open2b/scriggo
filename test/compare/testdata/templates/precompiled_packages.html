{# render #}

{% import "fmt" %}
{% import . "math" %}
{% import s "strings" %}

{{ fmt.Sprint(10, 20) }}

{% x := -123.1 %}
The absolute value of {{ x }} is {{ Abs(x) }}

{% for part in s.Split("this is a Scriggo template", " ") %}{{ part }},{% end for %}
