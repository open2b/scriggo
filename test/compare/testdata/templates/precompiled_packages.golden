10 20

The absolute value of -123.1 is 123.1

this,is,a,Scriggo,template,
